lustre/datadog_checks/lustre/check.py

```diff
@@ -204,10 +204,12 @@ def _run_command(self, bin: str, *args: str, sudo: bool = False) -> str:
         if bin not in self._bin_mapping:
             raise ValueError('Unknown binary: {}'.format(bin))
         bin_path = self._bin_mapping[bin]
-        cmd = f'{"sudo " if sudo else ""}{bin_path} {" ".join(args)}'
+        cmd = [bin_path, *args]
+        if sudo:
+            cmd.insert(0, "sudo")
         try:
             self.log.debug('Running command: %s', cmd)
-            output = subprocess.run(cmd, timeout=5, shell=True, capture_output=True, text=True)
+            output = subprocess.run(cmd, timeout=5, shell=False, capture_output=True, text=True)
```

Comment:
Do I understand correctly that `shell=False` is to avoid variable expansion?
Since False is the default, should we consider removing the explicit argument?